src/SampleLibraryZeroTouch/Examples/TransformableExample.cs

```diff
@@ -0,0 +1,101 @@
+﻿using Autodesk.DesignScript.Geometry;
+using Autodesk.DesignScript.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SampleLibraryZeroTouch.Examples
+{
+    public class TransformableExample : IGraphicItem
+    {
+        public Geometry Geometry { get; private set; }
+        private CoordinateSystem transform { get; set; }
+
+        //want to hide this constructor
+        [Autodesk.DesignScript.Runtime.IsVisibleInDynamoLibrary(false)]
+        public TransformableExample(Geometry geo)
+        {
+            Geometry = geo;
+            //initial transform is just at the origin
+            transform = CoordinateSystem.ByOrigin(0, 0, 0);
+        }
+
+     /// <summary>
+     /// An object which knows how to draw itself in the background preview and uses a transform to take 
```

Comment:
no, it's part of `ITransformable` which is applied to `RenderPackages` - I will turn this summary into a simple comment.